src/components/EventDateFilters.vue

```diff
@@ -1,62 +1,92 @@
 <template>
-  <div>
+  <div class='event-date-filters'>
     <h5 v-if="showTitle" class="filter-events-title">Date of event</h5>
     <div class="filter-events-date-picker">
-      <input ref="startDate" type="text" :value="startDate" />
-      <strong>to</strong>
-      <input ref="endDate" type="text" :value="endDate" />
+      <div class='date-picker-header'>
+        <input
+          autofocus
+          v-on:click="currentCalendar = 'startDate'"
+          ref="startDate"
+          type="text"
+          :value="startDate"
+        />
+        <strong>to</strong>
+        <input
+          v-on:click="currentCalendar = 'endDate'"
+          ref="endDate"
+          type="text"
+          :value="endDate"
+        />
+      </div>
+      <div id='calendar-container'>
```

Comment:
Is it possible here to use classes for the styles instead of ids, and if components need to be referenced give them a `ref`?

For example, `<div class="calendar-container" ref="container">` can then be referenced in the JS as `this.$refs.container`.